src/modules/Elsa.Workflows.Management/Filters/WorkflowInstanceFilter.cs

```diff
@@ -106,19 +111,47 @@ public IQueryable<WorkflowInstance> Apply(IQueryable<WorkflowInstance> query)
         if (filter.WorkflowSubStatus != null) query = query.Where(x => x.SubStatus == filter.WorkflowSubStatus);
         if (filter.WorkflowStatuses != null) query = query.Where(x => filter.WorkflowStatuses.Contains(x.Status));
         if (filter.WorkflowSubStatuses != null) query = query.Where(x => filter.WorkflowSubStatuses.Contains(x.SubStatus));
-        if (filter.ToCreatedAt is not null) query = query.Where(x => filter.ToCreatedAt >= x.CreatedAt);
-        if (filter.FromCreatedAt is not null) query = query.Where(x => filter.FromCreatedAt <= x.CreatedAt);
+        if (filter.HasIncidents != null) query = filter.HasIncidents == true ? query.Where(x => x.IncidentCount > 0) : query.Where(x => x.IncidentCount == 0);
+
+        if (TimestampFilters != null)
+        {
+            foreach (TimestampFilter timestampFilter in TimestampFilters)
+            {
+                var column = timestampFilter.Column;
```

Comment:
One of the properties of **TimestampFilter** can be null and have a bug for Eg:
with : 
?TimestampFilters=[{ "CreatedAt", "Operator": 1, "Timestamp": "2024-02-16T00:00:00"}]

Reply:
Should be good now with input validation 👍🏻